music/admin.py

```diff
@@ -17,4 +27,18 @@ def add_music(self, request, queryset):
             music.room.add_music(**MusicSerializer(music).data)
         return
 
+    def duplicate_music(self,request,queryset):
+        targetRoom = Room.objects.get(name=request.POST['nrroom'])
+        if targetRoom:
+            for music in queryset:
+                if targetRoom.name != music.room.name:
+                    new_entry = music
```

Comment:
Well if you're right, How do you explain that 2 entry are saved in the DB, with 2 different id and 2 different room ? 
To be sure i'll look at the DB directly, but i'm pretty confident that the performance is as expected